Adds screen reader only link to download Form 8 PDF, and another screen reader only link to skip iframe

amprokop opened the pull request:
Adds screen reader only content since our browser PDF viewer doesn't display the content in an accessible way.

Fixes #354

shanear commented on app/assets/stylesheets/_main.scss:
```diff
@@ -27,3 +27,17 @@
+// invisible onscreen but read out by screen readers.
+// Screen reader only content is not generally recommended,
+// so only use this to resolve major technical hurdles.
+.off-screen {
```
we already have a class that does this. `.usa-sr-only`

shanear replied:
🇺🇸

shanear commented on spec/feature/confirm_certification_spec.rb:
```diff
@@ -32,4 +32,13 @@
     expect(certification.reload.completed_at).to eq(Time.zone.now)
   end
+
+  scenario "Visiting certification page as a sighted user" do
```
I'm thinking this test should actually click the link, to make sure it downloads the pdf.

Also you can add this stuff into another test that's confirming the certification. Seems a little small to be in its own test.

amprokop commented:
@shanear — addressed your comments. i added more test logic. seems to me that there's enough weight in the new test to justify breaking it out into its own scenario, but i can move it back if you prefer! mind reviewing the changes?

shanear commented:
@amprokop agreed, sounds good. 👍
